.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
+    name: Webhint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Webhint
+        run: |
+          npm install --save-dev hint@6.x
+          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.hintrc
+      - name: Webhint Report
+        run: npx hint --telemetry=off .
+  stylelint:
+    name: Stylelint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```

Comment:
[ ] This is an inline comment 🙂

Reply:
- [ ] I think, this comment doesn't give sense. There is a comment you have written in the "Review changes" GitHub feature which says "There is an unused internal script in index.html. I think using an external script file is a good coding practice". You can make this an inline comment. 

```
Request Changes: :recycle: 

I think making a descriptive README.md file will make your project look professional. You can use Microverse README template. https://github.com/microverseinc/readme-template :slightly_smiling_face: 
I think it's better to give meaningful titles for your commit messages. Capitalizing and using imperative tense is a good practice for your comments. You can use this link for more information https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/
Linter is flagging errors. Please investigate and correct the errors. :slightly_smiling_face: 
There is an unused internal script in index.html. I think using an external script file is a good coding practice. :+1: :+1: 
``` 
- You only need to make changes on the inline comment. Your first Review Summary iwas great and keep it as it is.

Reply:
- [ ] OPTIONAL: I think, this comment doesn't give much sense. There is a comment you have written in the "Review changes" GitHub feature which says "There is an unused internal script in index.html. I think using an external script file is a good coding practice". You can make this an inline comment. Look at the below review summary you made in your first submission. 

------------------------------------------------
Request Changes: :recycle: 

I think making a descriptive README.md file will make your project look professional. You can use the Microverse README template. https://github.com/microverseinc/readme-template :slightly_smiling_face: 
I think it's better to give meaningful titles for your commit messages. Capitalizing and using imperative tense is a good practice for your comments. You can use this link for more information https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/
Linter is flagging errors. Please investigate and correct the errors. :slightly_smiling_face: 
There is an unused internal script in index.html. I think using an external script file is a good coding practice. :+1: :+1: 
---------------------------------------------------

- You only need to make changes to the inline comment. Your first Review Summary was great and keep it as it is.
- [ ] OPTIONAL: You can use tick boxes for the list of comments which helps students to mark resolved comments. Use ` -  [ ]` characters to do so.